cypress/support/utils.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+function createUUID() {
```

Comment:
I'm actually wondering why we need to generate a semi-legit UUID v4 for the app state if we're just faking things anyway. We're already intercepting many calls to Keycloak (see `cy.intercept()` returning a token and user info) so the state token doesn't really have to change or even be legit.

Reply:
Fair point. And if we do want a UUID why not just use uuidv4 :) there's an npm package

Reply:
good point! swapping to UUIDv4